### DISCUSSION TITLE
Panel collapse & Code Syntax coloring

### DISCUSSION THREAD

**Discussion by notbozho** [solodit/feedback#7] [New Feature Requests]
So, basically, **the first one** is adding a collapse button on each panel so it would look something like this:

_(bad examples I made in a few min with inspect element and figma :) )_
![image](https://github.com/solodit/feedback/assets/47002869/eac948b5-85ad-4513-a073-01091b901f9e)
![image](https://github.com/solodit/feedback/assets/47002869/2c411188-86b6-4615-a590-be18d28bada0)

, **and the second one** is self explanationary... syntax coloring on code blocks

**Comment by notbozho** [comment]
Also, another option is to add handles next to each panel so you can drag them and customize the width of each panel and if you make the width less than certain px just collapse it like the pictures above.

**Comment by hans-cyfrin** [comment]
Although we did not implement a custom resizing mechanism, I believe the current UI is still good enough with the fold/unfold feature.
The syntax coloring is fixed.
Thanks for the suggestion.
